### ISSUE TITLE
Crashing when doing circle payments

### ISSUE THREAD

**Issue by th3geek** [bitromortac/lndmanage#34]
Hi,

I've had this happen 3 or 4 times now when doing circle payments:
```

Traceback (most recent call last):
  File "./lndmanage.py", line 361, in <module>
    main()
  File "./lndmanage.py", line 317, in main
    dry=not args.reckless)
  File "/home/user/lndmanage/lib/rebalance.py", line 66, in rebalance_two_channels
    raise NoRouteError("Have tried this route already.")
lib.exceptions.NoRouteError: Have tried this route already.
```

Let me know if I can provide you with any addition info. 
Thanks!

edit: I realize its because its trying the same route twice, but shouldn't the program just exit and not stackdump?

**Comment by th3geek** [comment]
Actually I shouldn't have said "exit" above. I should have said "continue to try the next set of channels" This has been happening more and more, and if it happens at the beginning it won't continue on and try the next route and its got me stuck a couple times. I've also had some other crashes, but I haven't been keeping track. I'll try to post the next one I find. 

Thanks again!

**Comment by bitromortac** [comment]
Rebalancing and circular payments haven't received a lot attention the last weeks, but once I have finished the regtest integration testing I'll get back to that.

**Comment by bb2k16** [comment]
hi there,

I think am also experiencing similiar issue:

![image](https://user-images.githubusercontent.com/22429074/66334922-e370ca80-e931-11e9-8fe6-1dcdf69d4107.png)

Currently using:
LND 0.7.1
lndmanage 0.7.1
Windows 10


EDIT:
after upgrading to 0.8 for both LND and lndmanage, this issue seems to have happened less, but still happens.

**Comment by DrVecctor** [comment]
I have (almost) the same issue with lnd 0.10 and lndmanage 0.10
only now it says:

DuplicateRoute("Have tried this route already.")

**Comment by bitromortac** [comment]
This should be fixed by #81 with better error reporting.
